clash-lib/prims/commonverilog/GHC_Natural.json

```diff
@@ -0,0 +1,26 @@
+[
+  {
+    "BlackBox": {
+      "name": "GHC.Natural.naturalFromInteger",
+      "kind": "Expression",
+      "type": "naturalFromInteger :: Integer -> Natural",
+      "template": "$unsigned(~ARG[0][(~SIZE[~TYPO]-1):0])"
```

Comment:
you're only allowed to "slice" variables, so better to use `~VAR[XYZ][N]` construct instead of `~ARG[N]`

Reply:
I copied this from it's brother in `Clash_Sized_Internal_BitVector.json` and didn't catch it. Whoops. Should be fixed now.